charts/stable/domoticz/values.yaml

```diff
@@ -21,31 +20,25 @@ service:
         enabled: true
         targetPort: 6144
         port: 6144
-  comm2:
-    enabled: true
-    ports:
-      comm2:
-        enabled: true
-        targetPort: 1443
-        port: 1443
 workload:
   main:
     podSpec:
       containers:
         main:
           probes:
             liveness:
-              type: tcp
+              path: /
             readiness:
-              type: tcp
+              path: /
```

Comment:
you can omit paths, it will default to that
also startup will default to tcp!

So you can omit probes completely!